tests/series_only/scalar_index_test.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import annotations
+
+import narwhals.stable.v1 as nw
+
+np = nw.dependencies.get_numpy()
+pd = nw.dependencies.get_pandas()
```

Comment:
you can just import these directly here (though if you use `constructor_eager` they're probably not necessary)

Reply:
Thank you. I can confirm that `constructor_eager` works without them.